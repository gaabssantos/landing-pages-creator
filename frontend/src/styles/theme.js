export const theme = {
  mainBg: 'red',
  secondaryBg: 'blue',
};
