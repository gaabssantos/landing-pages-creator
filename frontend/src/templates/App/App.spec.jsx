import { screen } from '@testing-library/react';
import Home from '.';
import { renderTheme } from '../../styles/render-theme';

test('renders learn react link', () => {
  const { container } = renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', {
    name: 'Hello!',
  }).parentElement;
  expect(headingContainer).toHaveStyle({
    background: 'red',
  });
  expect(container.firstChild).toMatchSnapshot();
});
