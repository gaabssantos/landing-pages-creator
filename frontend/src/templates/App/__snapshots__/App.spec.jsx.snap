// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="App"
>
  <div
    class="sc-braxZu fsMZIU"
  >
    <h1>
      Hello!
    </h1>
  </div>
</div>
`;
